### PR TITLE
Error messages

### PR DESCRIPTION
**Pull request by KarineValenca**
Solves #21 
Screen Shot 2020-04-28 at 15 55 08

### CODE REVIEW

**Comment by eabili0** [comment]
@KarineValenca, I cannot rebase this PR. Could you please review your history and rebase with master?

File: README.md [hunk 1]
```diff
@@ -155,6 +155,25 @@ e.g. disabling
...
+    <param-value>error-info</param-value>
+</init-param>
+```
+
+Your java code should look like this:
+
+```java
+req.setAttribute("error-info", "Page not found");
+```
+
```
Comment (eabili0) [inline README.md:176]: @werberson, is this standard practice?

Comment (werberson) [reply README.md:176]: Yes, It is commonly used to share values in the request scope.

Comment (eabili0) [reply README.md:176]: Thank you @werberson!

**Review by eabili0 — APPROVED** [review]

**Comment by eabili0** [comment]
Thank you @KarineValenca ! Awesome work!

**Pull request merged**